flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/BootstrapToolsTest.java

```diff
@@ -423,4 +425,60 @@ public void testGetDynamicProperties() {
 		String dynamicProperties = BootstrapTools.getDynamicProperties(baseConfig, targetConfig);
 		assertEquals("-Dkey.b=b2 -Dkey.c=c", dynamicProperties);
 	}
+
+	/**
+	 * Remove 15% of the heap, at least 384MB.
```

Comment:
This description can not cover all the cases of this test. You can remove the description completely or give a more precise one.

Reply:
I have removed it.